packages/components/src/ListView/Items/Items.component.js

```diff
@@ -25,7 +46,7 @@ export class ItemsComponent extends React.PureComponent {
 
 		this.hasToggleAll = this.props.showToggleAll && this.props.items.length > 1;
 		this.cache = new CellMeasurerCache({
-			defaultHeight: props.getItemHeight(),
+			defaultHeight: props.getItemHeight ? props.getItemHeight() : ROW_HEIGHT,
```

Comment:
I would prefer a `defaultProps` `getItemHeight: () => ROW_HEIGHT;` ; WDYT ?

Reply:
If a default value is provided for getItemHeight, then old computation method will be used. I want to let the component autonomous regarding height computation when there is no method provided (as mentionned in the description, to avoid breaking changes).